src/js/relay.firefox.com/get_profile_data.js

```diff
@@ -44,7 +44,7 @@
     // Get the api token from the account profile page
     const profileMainElement = document.querySelector("#profile-main");
     const apiToken = profileMainElement.dataset.apiToken;
-    browser.storage.local.set({ apiToken });
+    await browser.storage.local.set({ apiToken });
```

Comment:
Technically this change wasn't needed for this bugfix, but I thought it'd be cleaner to do it here too.